app/src/main/java/com/zeapo/pwdstore/PasswordStore.kt

```diff
@@ -703,7 +702,7 @@ class PasswordStore : AppCompatActivity() {
                             continue
                         }
                         val destinationFile = File(target.absolutePath + "/" + source.name)
-                        val basename = FilenameUtils.getBaseName(source.absolutePath)
+                        val basename = FileUtils.getBaseName(source.absolutePath)
```

Comment:
if we're reimplementing these anyway I'd prefer to make `baseName` an extension property on `java.io.File`

Reply:
We use this over a string argument in `PgpActivity` so might not be feasible to make this change.